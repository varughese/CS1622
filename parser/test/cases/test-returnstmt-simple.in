int foo(void)
{
    return 1;
}

void main(void)
{
    return;
}
