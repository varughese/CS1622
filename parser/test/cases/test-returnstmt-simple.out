[program
  [fun-declaration
    [int]
    [foo]
    [params]
    [compound-stmt 
      [return-stmt [1]]
    ]
  ]
  [fun-declaration
    [void]
    [main]
    [params]
    [compound-stmt 
      [return-stmt]
    ]
  ]
]
